essence-of-live-coding-frp-test/essence-of-live-coding-frp-test.cabal

```
@@ -0,0 +1,62 @@
+name:                essence-of-live-coding-frp-test
+version:             0.1.0
+description:         Test framework for essence-of-live-coding-frp using Linear Temporal Logic
+homepage:            https://github.com/turion/essence-of-live-coding
+license:             BSD3
+license-file:        LICENSE
+author:              Ivan Perez, Manuel Bärenz, and Miguel Negrão
```

```suggestion
author:              Iván Peréz, Manuel Bärenz, and Miguel Negrão
```
And similar places elsewhere.